### DISCUSSION TITLE
asset value is zero

### DISCUSSION THREAD

**Discussion by DevAnyKR** [moneymanagerex/moneymanagerex#6617] [Q&A]
![image](https://github.com/moneymanagerex/moneymanagerex/assets/110871727/edbe5b13-78e7-462a-b4a9-fe46fbb6a77b)

ID no. 2 (자동차 = Car)

initial Value, Current Value is 0

and

![image](https://github.com/moneymanagerex/moneymanagerex/assets/110871727/e4e6c8f1-2ff6-4c60-82ac-c1f33a1055cb)

disabled 'Value' Input box. Why???

![image](https://github.com/moneymanagerex/moneymanagerex/assets/110871727/1125f57e-2c12-439c-ad3d-a15c3359a694)
The settings are the same as other assets.

![image](https://github.com/moneymanagerex/moneymanagerex/assets/110871727/3e7e997a-15af-472f-a559-b2ac8b2c4301)

Transactions exists.

Other assets will display their value normally.

Please let me know how to see the value of CAR asset.

**Comment by n-stein** [comment]
Right click on your "Car" asset and select "View asset transactions". Do they net to 0?

**Reply by DevAnyKR** [reply]
![image](https://github.com/moneymanagerex/moneymanagerex/assets/110871727/f0eca520-c9f0-4382-8331-14ec0cc1ca04)

**Reply by DevAnyKR** [reply]
![image](https://github.com/moneymanagerex/moneymanagerex/assets/110871727/d7183ec5-ba2a-41d7-acf0-52952e6696d1)
The same is if set the Value change to None, % Rate to 0.

Value component is disabled.

**Reply by DevAnyKR** [reply]
Is this symptom intended by the developer or is it a bug?
I just want to know why it's zero and why it can't be changed.

**Reply by DevAnyKR** [reply]
![image](https://github.com/moneymanagerex/moneymanagerex/assets/110871727/508fe8ad-4038-428f-b5e1-1c1a4eed1d9c)

I clicked on the page while continuing to move, and I was able to change the value, but now the contents don't appear in the dialog box.
🤣🤣🤣
